pyinfra/api/util.py

```diff
@@ -352,9 +355,12 @@ def make_command(
             ' '.join(su_bits), su_user, shell_executable, command,
         )
 
-    # Otherwise just sh wrap the command
     else:
-        command = '{0} -c {1}'.format(shell_executable, command)
+        # Otherwise just sh wrap the command
```

Comment:
We can undo all the changes to this file and not use `make_command` at all!

Reply:
done